### DISCUSSION TITLE
Payment Request Interface

### DISCUSSION THREAD

**Discussion by juandelcid** [btcpayserver/btcpayserver#2791] [Ideas / Feature Requests]
Hi guys!

The current flow for creating a Payment Request seems a little cumbersome to me:

1. Create a payment request
2. Fill in the form and save
3. View
4. Copy the Link
5. Email the client (using your own email client)

Can we work on a way to reduce the steps?

Here's what I have in mind:

1. Create a payment request
2. Fill in the form including an email for the intended recipient
3. Save and send

Also, there should be a way to upload a CSV file with billing data (title, amount, text, email, etc.) , have the server create the corresponding Payment Requests and send the emails.

Thanks for your time,
-Juan Gabriel

**Comment by pavlenex** [comment]
cc @dstrukt @dennisreimann may be good topic for the next design meet :)

**Reply by dstrukt** [reply]
Yeah, we can add this to the agenda - glad to see the Payment Request feature is being used, could definitely be refined.
